### DISCUSSION TITLE
Break out filename and file path in the Videos categories

### DISCUSSION THREAD

**Discussion by olinesn** [talmolab/sleap#1855] [Ideas]
Could there be an option to separate the video and filepath columns in the Videos tab? Right now, the entire path plus file name can be pretty long, when sometimes I'm just looking for the first few characters of a file name.

Thanks!
Stefan

**Comment by 7174Andy** [comment]
Implemented in https://github.com/talmolab/sleap/pull/2052

**Comment by roomrys** [comment]
Long awaited, but finally integrated. This issue has been fixed in the SLEAP 1.4.1 release!

**Reply by olinesn** [reply]
Thanks!!
